types/react-native-snap-carousel/index.d.ts

```diff
@@ -172,7 +230,38 @@ export interface CarouselStatic extends React.ComponentClass<CarouselProps> {
     snapToPrev(animated?: boolean): void;
 }
 
-export type CarouselProperties = ScrollViewProperties & CarouselProps & React.Props<CarouselStatic>;
+export type CarouselProperties<T> = ScrollViewProperties & CarouselProps<T> & React.Props<CarouselStatic<T>>;
```

Comment:
Could you add an example / test for the ParallaxImage to `react-native-snap-carousel-tests.tsx`?

Reply:
Added a `SnapCarouselWithParallaxTest` example